TYPES.md

```diff
@@ -63,14 +63,15 @@ Parameters are :
      - it's used to generate actions types
      - it's used to retrieve informations from selectors
    - **prefix** (optional) is added to actions to avoid some collisions when there are two reducers with the same name in two distincts paths
+   - **defaultData** (optional), set the default data value, used by `reset` action and for initialisation (default is an empty hashmap model)
 
 ### state
 ```es6
 {
   data: { <key1>: <instance1>, <key2>: <instance2> },
   array: [<instance1>, <instance2>],
   keys: [<key1>, <key2>],
-  initialized: true,
+  initialized: false,
```

Comment:
What about: `<true|false>`

Reply:
by default it's `false`

Reply:
you think `initialized: <true|false>` is better for understanding ?

Reply:
this is not the default state, this is about the state shape

Reply:
so  `initialized: <boolean>` ?

Reply:
ok